src/precice/SolverInterface.hpp

```diff
@@ -498,87 +498,66 @@ class SolverInterface {
       int *         ids) const;
 
   /**
-   * @brief Sets mesh edge from vertex IDs, returns edge ID.
+   * @brief Sets a mesh edge from vertex IDs
    *
    * @param[in] meshID ID of the mesh to add the edge to
    * @param[in] firstVertexID ID of the first vertex of the edge
    * @param[in] secondVertexID ID of the second vertex of the edge
    *
-   * @return the ID of the edge
-   *
    * @pre vertices with firstVertexID and secondVertexID were added to the mesh with the ID meshID
    */
-  int setMeshEdge(
+  void setMeshEdge(
       int meshID,
       int firstVertexID,
       int secondVertexID);
 
   /**
-   * @brief Sets mesh triangle from edge IDs
+   * @brief Sets multiple mesh edge from vertex IDs
    *
-   * @param[in] meshID ID of the mesh to add the triangle to
-   * @param[in] firstEdgeID ID of the first edge of the triangle
-   * @param[in] secondEdgeID ID of the second edge of the triangle
-   * @param[in] thirdEdgeID ID of the third edge of the triangle
+   * @param[in] meshID ID of the mesh to add the edges to
+   * @param[in] size the amount of edges to set
+   * @param[in] vertices an array containing 2*size vertex IDs
    *
-   * @pre edges with firstEdgeID, secondEdgeID, and thirdEdgeID were added to the mesh with the ID meshID
+   * @pre vertices were added to the mesh with the ID meshID
    */
-  void setMeshTriangle(
-      int meshID,
-      int firstEdgeID,
-      int secondEdgeID,
-      int thirdEdgeID);
+  void setMeshEdges(
+      int        meshID,
+      int        size,
+      const int *vertices);
```

Comment:
I think it would be good to give an example here. The necessary ordering of vertices is not yet clearly described. "l1, r1, l2, r2, l3, r3" vs "l1, l2, l3, r1, r2, r3" 
Same concern for all other bulk variants below.

Reply:
Good point!